clowdapp.yaml

```diff
@@ -38,6 +38,8 @@ objects:
     - name: import-ssg
       schedule: "*/5 * * * *" # every 5 minutes
       concurrencyPolicy: Forbid
+      annotations:
+        ignore-check.kube-linter.io/no-readiness-probe: "This job does not need a readiness probe because it does not handle incoming traffic."
```

Comment:
Do you have a link to docs on the `ignore-check.kube-linter.io/no-readiness-probe` annotation at hand? Thanks.

Reply:
> Rake is the main process of the container. I do not think that checking its existence does any benefit. It is bound to the container, so whenever it terminates the container will terminate with it. Liveness probe could rather check progress of the jobs, checking that they are not stuck waiting indefinitely.

@vkrizan we can't really do that easily :laughing: that's why I proposed what I proposed to go around this. We could cook up some DRB magic but I don't really see the point :disappointed:

Reply:
Yeah, here's the link that Luis sent me (https://www.civo.com/learn/yaml-best-practices-using-kubelinter#ignoring-a-check)

I was going to ask if you guys thought this was correct or not too. It's a little confusing to test